### ISSUE TITLE
It shouldn't log the credentials in the console.

### ISSUE THREAD

**Issue by kaviansson** [Steveorevo/node-red-contrib-credentials#7]
Anytime to open the node for edit, it logs all properties and their values to the browsers console.

Please kindly see the screenshot 
![image](https://github.com/Steveorevo/node-red-contrib-credentials/assets/71076920/007b0269-d8c6-4741-9efa-3e57bbb1e303)

**Comment by Steveorevo** [comment]
Agreed. There is an implicit console.log that should be removed. Granted, a savvy user can still use a DOM inspector; but we shouldn't be writing to the console. Will create a .3 release and include this.
